src/stores/appInterface.ts

```diff
@@ -46,6 +52,24 @@ export const useAppInterfaceStore = defineStore('responsive', {
         .padStart(2, '0')
       this.UIGlassEffect.bgColor = `#${hex}${alphaHex}`
     },
+    setHighlightedComponent(component: string) {
+      this.componentToHighlight = component
+    },
+    setMainMenuVisibility(value: boolean) {
+      this.mainMenuVisibility = value
+    },
+    setMainMenuStep(step: number) {
+      this.mainMenuCurrentStep = step
+    },
+    setCurrentConfigComponent(componentIndex: number) {
+      this.configComponent = componentIndex
+    },
+    setKeepGlassModalAlwaysOnTop(value: boolean) {
+      this.isGlassModalAlwaysOnTop = value
+    },
+    setTutorialVisibility(value: boolean) {
+      this.isTutorialVisible = value
+    },
```

Comment:
Are those methods needed?

One of Pinia's advantages is that it doesn't need mutation methods, and one can easily set the values from both inside and outside the store directly.

Unless we need the methods to create side-effects (like we do when we set the widgets view index for example, where we need to confirm if the index is valid), I think we can get rid of them.

Reply:
You are correct.
Those methods should exist in Pinia, only when some logic has to be applied on a state change.
Changing that..